src/alacris_lib/operator_contract.mli

```diff
@@ -13,23 +13,23 @@ val topic_of_amount:   TokenAmount.t -> Bytes.t option
 val topic_of_hash:     Digest.t      -> Bytes.t option
 
 
-val retrieve_contract_address_quadruple : Digest.t -> (Address.t * Digest.t * Digest.t * Revision.t) Lwt_exn.t
+type contract_address_config =
+  { contract_address : Address.t
+  ; code_hash : Digest.t
+  ; creation_hash : Digest.t
+  ; creation_block : Revision.t
+  }
 
 
-val set_contract_address : Address.t -> unit
-(** set the address of the contract on Ethereum
-    TODO: use a fixed address, obviating this call (?) *)
+val retrieve_contract_address_quadruple : Digest.t -> contract_address_config Lwt_exn.t
```

Comment:
Replace `contract_address_quadruple` and `contract_address_config` by just `contract_config` ?

Reply:
Renaming has been done